health/warnings.go

```diff
@@ -111,6 +113,7 @@ var noDERPHomeWarnable = Register(&Warnable{
 	DependsOn:           []*Warnable{NetworkStatusWarnable},
 	Text:                StaticMessage("Tailscale could not connect to any relay server. Check your Internet connection."),
 	ImpactsConnectivity: true,
+	TimeToVisible:       10 * time.Second,
```

Comment:
This doesn't impact the logging to stderr, or does it?

Reply:
yes it does, as discussed